data/brands/amenity/fast_food.json

```
@@ -9108,6 +9109,22 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "עגבניה",
+      "id": "agvania-dbdcb8",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "amenity": "fast_food",
+        "cuisine": "pizza",
+        "delivery": "yes",
+        "name": "עגבניה",
+        "name:en": "Agvania",
+        "name:he": "עגבניה",
+        "operator": "Agvania Pizza",
+        "operator:he": "Agvania Pizza",
```

```suggestion
        "operator:en": "Agvania Pizza",
```
Look like this name is in English?